README.md

```diff
@@ -48,34 +48,6 @@ interface Option {
 }
 ```
 
-## React Usage
```

Comment:
We can, but it requires more build configuration so that we export the `iframe/App.tsx` along with the rest of the package.

Reply:
we can re-expose this as a separate PR. Reference: 

> The React embed usage is no longer viable for the time being - but this can be fixed relatively easily by exposing iframe/app.tsx as a package later.